athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/handlers/MetadataHandler.java

```diff
@@ -115,6 +120,10 @@
     private final String spillBucket;
     private final String spillPrefix;
     private final String sourceType;
+    private static BucketState bucketState;
```

Comment:
Why static? If this is because of something I said, that is my mistake. I think it is ok for this to be instance specific.

Reply:
OK, I'll make the changes